### PR TITLE
add instructions to train FLAVA model

### PR DESCRIPTION
**Pull request by edward-io**
Summary:

Test plan:

Fixes #{issue number}

### CODE REVIEW

File: examples/flava/native/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Usage Instructions
+
+## Prerequisites
+
+- Installed PyTorch 1.13 or nightly
```
Comment (langong347) [inline examples/flava/native/README.md:5]:
```suggestion
- Install PyTorch 1.13 or nightly 
```

File: examples/flava/native/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Usage Instructions
+
+## Prerequisites
+
+- Installed PyTorch 1.13 or nightly 
+- `git clone https://github.com/facebookresearch/multimodal`
```
Comment (ankitade) [inline examples/flava/native/README.md:6]: just link to the library readme as first step and pip install on the flava requirements as second step? that should be enough?

File: examples/flava/native/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Usage Instructions
+
```
Comment (ankitade) [inline examples/flava/native/README.md:2]: Can we add some description here i.e. this is a native pytorch lightweight implementation to run scaling studies which is subset of original code. For original code , people should follow the other script in the FLAVA directory

**Comment by ankitade** [comment]
also add instructions on how to override the configs we care about, i.e. strategy, activation ckpting and bsz

File: examples/flava/native/README.md (hunk 2 above)
Comment (edward-io) [reply examples/flava/native/README.md:6]: the examples/ folder isn't included if users install the library via pip:

https://github.com/facebookresearch/multimodal/blob/main/setup.py#L80

Comment (ankitade) [reply examples/flava/native/README.md:9]: you need to link to this https://github.com/facebookresearch/multimodal/blob/main/README.md#building-from-source

File: examples/flava/native/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+This is a lightweight native pytorch implementation to run scaling studies on the FLAVA model. The original code is located at: [`examples/flava/train.py`](https://github.com/facebookresearch/multimodal/blob/main/examples/flava/train.py)
+
+## Prerequisites
+
+- Install PyTorch 1.13 or nightly 
+- Install [torchmultimodal library](https://github.com/facebookresearch/multimodal#installation)
```
Comment (ankitade) [inline examples/flava/native/README.md:8]: not needed? lib readme will cover

File: examples/flava/native/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+- `git clone https://github.com/facebookresearch/multimodal`
+- `cd multimodal/examples`
```
Comment (ankitade) [inline examples/flava/native/README.md:10]: still need to do pip install of flava requirements to get HF dependencies

File: examples/flava/native/README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+# Usage Instructions
+
+This is a lightweight native pytorch implementation to run scaling studies on the FLAVA model. The original code is located at: [`examples/flava/train.py`](https://github.com/facebookresearch/multimodal/blob/main/examples/flava/train.py)
+
+## Prerequisites
+
+- Install PyTorch 1.13 or nightly
```
Comment (ankitade) [inline examples/flava/native/README.md:7]: you dont need this line?

File: examples/flava/native/README.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+- `cd multimodal/examples`
+- `pip install -r flava/requirements.txt`
+
+## Training 
+
+### Configuration
+
+Configuration presets can be found at: `examples/flava/native/configs`
+
+Some config settings that are relevant for scaling: `batch_size`, `activation_checkpointing`, `strategy`.
```
Comment (ankitade) [inline examples/flava/native/README.md:18]:
```suggestion
Some config settings that are relevant for scaling: (local)`batch_size`, `activation_checkpointing`, `strategy`.
```

File: examples/flava/native/README.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+Configs can be overridden through command line, for example: `python -m flava.native.train config=flava/native/configs/2.7b.yaml training.batch_size=8 training.enable_amp=True training.activation_checkpointing=True training.strategy=fsdp`
```
Comment (ankitade) [inline examples/flava/native/README.md:20]:
```suggestion
Configs can be overridden through command line, for example: `python -m flava.native.train config=flava/native/configs/pretrain_debug.yaml training.batch_size=8 training.enable_amp=True training.activation_checkpointing=True training.strategy=fsdp`
```

**Comment by ankitade** [comment]
linter is complaining ?

**Review by ankitade — APPROVED** [review]

**Pull request closed without merging**